Experimental Setup
Basically microarray is a solid base having grid of spots where genetic ma- terial of known sequence is arranged systematically. It is mostly made up of glass on which single stranded DNA molecules are attached at fixed po- sitions. The size of the arrays can vary from microscope slide to square silicon chips. On an array, there can be thousands of spots and each spots contain number of identical DNA molecules. The microarray fabrication can be done in two ways: i) cDNA and ii) oligonucleotide.

Preparation of probe DNA
For the study of large-scale expression, a specific DNA sequence is needed for all genes whose expression values are to be measured. The selection of probe is done on the basis of resources available for obtaining the representation of the genes under studies. The simplest way is to amplify every known ORF in the genome and use it as a probe. PCR is used for the amplification purpose and allows multiplication of DNA fragments by millions in just few hours. ESTs may be used to identify distinct mRNA transcripts.

Printing array
In cDNA array, arrays are mostly printed on Poly-L-lysine coated glass microscope slide using arraying robot. During the arraying operation, a large number of slides are placed on and secured to a platter. The samples of DNA are placed in microliter plates on the stand. The reservoir slot of each tip is filled with 1 liter of DNA solutions. The tips are then lightly tapped at identical positions on each slide leaving a small drop of DNA solution on the poly-L-lysine coated slide. In Oligonucleotide array, oligos are printed at the spots instead of cDNA. Same robotics can be applied to manufacture both types of arrays. However, the preparation of oligonucleotide array is quite different. During fabrication of array, the probes are synthesized on the chip using photolithography.

Post processing of slides
This step consists of Rehydration and Blocking. The spots on the microarray are rehydrated to distribute DNA more evenly. In the blocking process, free reactive groups on the slide surface are modified to minimize their ability to bind to labeled target DNA. If these groups are not blocked, the labeled DNA target can bind to the surface of the slide.

Preparation of target
In this step, isolate mRNA from the samples and purify it. Since, mRNA degrades very fast; hence it is reverse-transcribed into more stable cDNA.

Hybridization
In hybridization process, a single stranded DNA molecule is bound to an- other single strand DNA molecule with a precisely matching sequence. After hybridization process, the microarray properly washed to eliminate any ex- cess labeled sample and finally dried using a centrifuge. Sometimes two types of target mRNA samples are simultaneously hybridized on the array, called two channel microarray experiment. In that case, two types of molecules are added to targets and uses fluorescent dyes like Cy3 and Cy5, which can be separated spectrally. The Cy3 is green and Cy5 is red when excited by laser light at specific wavelength.

Slide imaging
Under this step, the microarray is scanned to measure the fluorescent signal emitted at every spot that determine the amount of labeled sample bound to each spot. The laser scanning confocal microscope is used for this pur- pose. For single channel array, the array is scanned once but for two-channel experiment, it is scanned in two phases. In the obtained image, the inten- sity of each spot is proportional to the amount of mRNA from the sample, matching cDNA sequence of given pot. A gene expressed in a sample labeled with red dye and not expressed in the other sample will produce a red spot and vice versa.